actuators/cluster.go

```diff
@@ -17,39 +17,34 @@ limitations under the License.
 package actuators
 
 import (
-	"fmt"
-
+	"github.com/go-logr/logr"
 	"sigs.k8s.io/cluster-api-provider-docker/kind/actions"
 	clusterv1 "sigs.k8s.io/cluster-api/pkg/apis/cluster/v1alpha1"
 )
 
 // Cluster defines a cluster actuator object
 type Cluster struct {
-}
-
-// NewClusterActuator returns a new cluster actuator object
-func NewClusterActuator() *Cluster {
-	return &Cluster{}
+	Log logr.Logger
 }
 
 // Reconcile setups an external load balancer for the cluster if needed
 func (c *Cluster) Reconcile(cluster *clusterv1.Cluster) error {
 	elb, err := getExternalLoadBalancerNode(cluster.Name)
 	if err != nil {
-		fmt.Printf("%+v\n", err)
+		c.Log.Error(err, "Error getting external load balancer node")
```

Comment:
Similar to the `Info` calls, this should be

```
c.Log.Error("could not get external load balancer node", "stacktrace", fmt.Sprintf("%+v", err))
```

edit: it kind of sucks that we have to do this but because cluster-api Uses glog/klog...and we don't register glog/klog, we don't get any cluster-api logging output so we try and compensate by printing these error messages at every `return err` call site.

Reply:
@chuckha Not quite following. The interface for `Logger.Error` is this:
`func (logr.Logger).Error(err error, msg string, keysAndValues ...interface{})`
Where the error is a parameter in this function.

Reply:
resolved confusion offline. Keeping the Error message as is.